src/QmlControls/SettingsGroupLayout.qml

```diff
@@ -18,6 +18,7 @@ ColumnLayout {
     property string heading
     property string headingDescription
     property bool   showDividers:       true
+    property bool   showBorder:         true
```

Comment:
I'd leave this as is for now. Don't want to create a new UI paradigm of these group layout without borders until it is really needed. That is already mostly already covered by the SectionHeader control.